docs/documentation/api/Game.md

```diff
@@ -10,6 +10,12 @@
   // passed through the Game Creation API.
   setup: (ctx, setupData) => G,
 
+  // Optional function to validate the setupData before
+  // matches are created. If this returns a non-null
+  // value, an error will be reported to the user and
+  // match creation is aborted.
+  validateSetupData: (setupData, numPlayers) => null,
```

Comment:
If `validateSetupData` is expected to return a string if there is and error and `undefined` otherwise, perhaps it would be helpful for this example to show it returning an error string, something like:

```suggestion
  validateSetupData: (setupData, numPlayers) => 'setupData is not valid!',
```

Reply:
Updated.